build.rs

```diff
@@ -47,17 +47,18 @@ fn run() -> io::Result<()> {
             #[test]
             fn #test_name() {
                 use std::env;
+
```

Comment:
was this intended? I'm okay with it if it is, i don't feel super strongly

Reply:
Yeah, wanted to split stdlib vs. extern crate imports